kogito-codegen-modules/kogito-codegen-core/src/main/java/org/kie/kogito/codegen/core/io/CollectedResourceProducer.java

```diff
@@ -127,6 +147,39 @@ private static CollectedResource toCollectedResource(Path basePath, File file) {
     private static CollectedResource toCollectedResource(Path basePath, String resourceName, Resource resource) {
         resource.setResourceType(determineResourceType(resourceName));
         return new CollectedResource(basePath, resource);
+    }
+
+    private static boolean shouldIgnoreHiddenFiles() {
+        return Boolean.parseBoolean(System.getProperty(IGNORE_HIDDEN_FILES_PROP, "true"));
```

Comment:
I suggest changing things a bit, lets call the propery USE_HIDDEN_FILES, with a default value of false you can write

```suggestion
        return Boolean.getBoolean(USE_HIDDEN_FILES_PROP);
```

And the condicion looks like
```
p -> p.isFile() && (useHiddenFiles() || !p.isHidden())
```

Reply:
I agree that negate a property is more confusing, but "USE" not sure it's the right verb here, I'll think about it.